packages/fork-choice/src/protoArray/protoArray.ts

```diff
@@ -143,6 +147,7 @@ export class ProtoArray {
 
         // back-propagate the nodes delta to its parent
         deltas[parentIndex] += nodeDelta;
+        if (nodeBoost) boosts[parentIndex] = nodeBoost;
```

Comment:
this is also good option to handle boost, however I still prefer lighthouse's implementation https://github.com/sigp/lighthouse/blob/9ed92d6e7830893670e2176a4e3bfd571f344cbf/consensus/proto_array/src/proto_array.rs#L192
+ memory: it does not need another array
+ performance: it does not need another get/set from/to the array

the difference maybe not a lot through. I'd like to know your ideas @dapplion @wemeetagain

Reply:
Agree to go for maximum simplicity here

Reply:
:+1: